src/stream_engine.cpp

```diff
@@ -784,19 +784,6 @@ int zmq::stream_engine_t::next_handshake_command (msg_t *msg_)
 
         if (rc == 0)
             msg_->set_flags (msg_t::command);
-#ifdef ZMQ_BUILD_DRAFT_API
```

Comment:
I see, thanks.
Does the multiple event change means that there is a public API change?